docs/git-node.md

```diff
@@ -330,7 +330,7 @@ ncu-config set waitTimeMultiApproval 48
 
 ## `git node v8`
 
-Update or patch the V8 engine.  
+Update or patch the V8 engine.
```

Comment:
Was this change on purpose? IIUC those spaces are here to tell the Markdown interpreter there should be a line break.

Reply:
Possibly it was caused by `npm run lint-fix`

Reply:
Let's add them back and see if the linter complains
```suggestion
Update or patch the V8 engine.  
```